src/docfx/Program.cs

```diff
@@ -92,7 +92,12 @@ internal static int ExecSubCommand(string[] args)
                 command.Exec(context);
                 return 0;
             }
-            catch (Exception e) when (e is DocumentException || e is DocfxException)
+            catch (Exception e) when (e is DocumentException)
+            {
+                Logger.LogError(e.Message, code: ErrorCodes.Build.DocumentError);
+                return 1;
```

Comment:
would this cause duplicate error logs?

Reply:
Yes it will... This PR only makes this line system error, but the code inside is still system error as it has no code assigned... Let me fix it.

Reply:
After second thoughts, I kept the outer `LogError`, but `DocumentException` can carry information like `Code`. I have to update all the code that throws `DcumentException` if I had kept the inner `LogError`...

If this is good, Sharanya's PR can use the same pattern to throw `DocumentException`.

Reply:
I see... Would the duplicate error logs be a problem?